portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1385,10 +1375,10 @@ definition {
 			virtualHost = "www.baker.com",
 			webId = "www.baker.com");
 
-		User.firstLoginPG(
+		User.firstLoginUI(
 			password = "test",
-			userEmailAddress = "test@www.able.com",
-			virtualHostsURL = "http://www.able.com:9080");
+			specificURL = "http://www.able.com:9080",
+			userEmailAddress = "test@liferay.com");
```

Comment:
The first login email should be test@www.able.com, otherwise the user will not display under user admin. See testray result: https://testray.liferay.com/home/-/testray/case_results/1532525027

Reply:
Okay, I modify this email address.